src/deepsparse/v2/routers/router.py

```diff
@@ -32,20 +33,32 @@ class Router:
 
     """
 
-    def __init__(self, end_route: Union[str, int], start_route: Union[str, int]):
+    def __init__(
+        self,
+        end_route: Union[str, int],
+        start_route: Union[str, int],
+        route: Union[List, Dict],
+    ):
         self.START_ROUTE = start_route
         self.END_ROUTE = end_route
```

Comment:
Would it make sense to make END_ROUTE a sentinel operator rather than an actual operator index?

Reply:
No it's good my bad